docs/OV_Runtime_UG/migration_ov_2_0/deployment_migration.md

```diff
@@ -0,0 +1,156 @@
+# Install & Deployment {#openvino_2_0_deployment}
+
+## Introduction
+
+"Easy to use" is one of the main points for OpenVINO 2.0 concept. This includes not only simplification of migration from frameworks to OpenVINO Toolkit, but also how OpenVINO is organized, how the development tools are used, how to develop and deploy OpenVINO-based applications.
+
+Starting from OpenVINO 2.0, Model Optimizer, Post Training Optimization tool, Open Model Zoo tools and other Python-based Development tools are distributed via [PyPI](https://pypi.org/project/openvino-dev/) only. This simplifies user experience, because in older versions of OpenVINO development tools should be installed and used differently dependning on distribution type (debian packages vs. PyPi).
```

Comment:
I think will be good to provide an example, how openvino-dev package can be installed.
`pip install openvino-dev`